OpenMLDB/src/src/main/java/com/_4paradigm/benchmark/OpenMLDBPerfBenchmark_Cluster.java

```diff
@@ -205,47 +219,65 @@ public void load2(String folderPath) {
             Util.executeSQL("SET @@execute_mode='online';", executor);
             String loadDataSQL = "LOAD DATA INFILE '"+folderPath + "benchmark' INTO TABLE benchmark options(format='parquet', header=true, mode='append');";
             System.out.println(loadDataSQL);
-            Util.executeSQL_block(loadDataSQL, executor);
+            if (!Util.executeSQL_block(loadDataSQL, executor)) {
```

Comment:
Why not add a method like:
```
onlineLoad(path, table):
    log.info()
    loadSQL = ....path .... table
    executeSQL_block()
```
I think `executeSQL_block` throws the exception is better than just a boolean returned value.